src/pages/zh-cn/guides/integrations-guide.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: 使用集成
+---
+
+使用 **Astro 集成**只需几行代码就能为你的项目添加新的功能和行为。你可以自己编写一个自定义的集成，或者从 [npm](https://www.npmjs.com/search?q=keywords%3Aastro-component&ranking=popularity) 获取流行集成。
+
+
+- 解锁 React、Vue、Svelte、Solid 和其他流行的 UI 框架。
+- 只需几行代码就能整合 Tailwind 和 Partytown 等工具。
+- 为你的项目添加新功能，如自动生成网站地图。
+- 编写自定义代码，与构建过程、开发服务器等挂钩。
+
+:::note[实验性]
+目前只有官方的Astro集成（那些发布在npm上的`@astrojs/`）被默认支持，以保护用户不被破坏。
+
+**启用第三方集成：**使用 `--experimental-integrations` CLI 标志运行 Astro，或者在 Astro 配置文件中包括 `experimental: { integrations: true }`。
+:::
+
+## 教程：在项目中添加 React
+
+在这个例子中，我们将添加 `@astrojs/react` 集成，好让你的 Astro 项目支持 React。添加任何其他框架（Preact、Vue、Svelte 或 Solid.js）的过程几乎都是相同的，可以使用下面概述的相同步骤。
+
+:::tip[快速开始]
+Astro 提供了 `astro add` 命令，你可以运行它自动完成添加官方 Astro 集成这一过程。你可以运行 `npx astro add react`，而不是下面的步骤。
+
+跳到[自动集成设置](#自动集成设置)了解更多细节。
+:::
+
+首先，你需要安装集成和任何你希望在项目中使用的相关包。对于 React 来说，你需要安装 `@astrojs/react`集成***和*** `react` + `react-dom` 包。
+
+```bash
+npm install --save-dev @astrojs/react
+```
+
+当包安装完成后，在你的 `astro.config.mjs` 项目配置文件中添加两行新内容。
+
+```diff
+  // astro.config.mjs
+  import { defineConfig } from 'astro/config';
++ import react from '@astrojs/react';
+
+  export default defineConfig({
++   integrations: [react()],
+  });
+```
+
+第一行是导入语句，将集成导入到你的配置文件中。第二行调用集成函数（`react()`）并添加集成，Astro 就知道要使用它了。
+
+这就是了! 重新启动 Astro，新的集成应该立即生效。
+
+如果你在启动时看到一个错误，请确保你：
+
+- ✅ 使用 npm 安装了所需的包
+- ✅ 在 `astro.config.mjs` 文件中导入了集成
+- ✅ 以函数形式调用你的集成（`[react()]`，而不是 `[react]`）。
+- ✅ 删除废弃 `renderers:` 配置（v0.25 前曾使用）。
+
+## 自动集成设置
+
+Astro 最近推出了**实验性的** `astro add` 命令来自动设置集成。
+
+:::caution
+在更新你的任何文件之前，我们都会要求你进行确认，但为了以防万一，请做好备份。
+:::
+
+不需要上述的手动配置，只需运行 `astro add [name]`，我们的自动集成向导将更新你的配置文件并安装任何必要的依赖。
+
+```shell
+# Using NPM
+npx astro add react
+# Using Yarn
+yarn astro add react
+# Using PNPM
```

Comment:
I think we should translate “Using” in these comments. I’ve made a suggestion based on a machine translation, but please use whatever would read best!

```suggestion
# 用 npm
npx astro add react
# 用 Yarn
yarn astro add react
# 用 pnpm
```